Makefile-libostree.am

```diff
@@ -156,7 +156,15 @@ libostree_1_la_LIBADD = libotutil.la libglnx.la libbsdiff.la libostree-kernel-ar
 libostree_1_la_LIBADD += $(bupsplitpath)
 EXTRA_libostree_1_la_DEPENDENCIES = $(top_srcdir)/src/libostree/libostree.sym
 
-EXTRA_DIST += src/libostree/libostree.sym
+EXTRA_DIST += \
+	src/libostree/libostree.sym \
+	src/libostree/libostree-experimental.sym \
```

Comment:
I’m not sure I want to commit to that level of stability for the unstable API. I don’t think we discussed a ‘separate `.so`’ option — I’m not sure how it would work, given that the experimental functions need access to the normal `libostree.so` internals.